tensornetwork/backends/base_backend.py

```diff
@@ -334,24 +334,31 @@ def eigh(self, matrix: Tensor):
 
   def eigs(self,
            A: Callable,
+           args: List,
```

Comment:
We should have the default of `args` be `None` and then have the check

```
if args is None:
  args = []
```

That allows us to avoid the annoying `eigs(matvec, args=[])`

Reply:
that's true, I'll fix it